src/service/comparison_vs_peers/comparison_vs_peers_service.py

```diff
@@ -24,6 +24,22 @@ def remove_base_metrics(self, company: dict) -> None:
         for metric in base_metrics:
             company.pop(metric, None)
 
+    def anonymized_companies(
```

Comment:
You have a similar function on the investment year report file, could you please move it to the calendar report_

Reply:
it was moved